rules/prettier.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": ["prettier"],
+  "plugins": ["prettier"],
+  "rules": {
+    "prettier/prettier": [
+      "error",
+      {
+        "singleQuote": true,
+        "tabWidth": 4,
+        "printWidth": 80
+      }
+    ],
+    "react/jsx-sort-props": 0,
```

Comment:
I think we do want this?

Reply:
whoops probably should not be any rule overrides in here